src/modules/dome.c

```diff
@@ -121,6 +121,26 @@ WINDOW_getFps(WrenVM* vm) {
   wrenSetSlotDouble(vm, 0, debug->avgFps);
 }
 
+internal void
+WINDOW_setColor(WrenVM* vm) {
+  ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
+  ASSERT_SLOT_TYPE(vm, 1, NUM, "color");
+  uint32_t color = (uint32_t)wrenGetSlotDouble(vm, 1);
+  uint8_t r = color & 0xFF;
+  uint8_t g = (color & (0xFF << 8)) >> 8;
+  uint8_t b = (color & (0xFF << 16)) >> 16;
```

Comment:
This decoding could be refactored into a function which sets these properties, and then can be used here and in `image.c`.

Reply:
I think exactly like this and the previous comment, just not wanted to touch the existing code 😉.

Reply:
Where should this function be placed?

Reply:
Actually, it's inside engine.c (`ENGINE_pset()`).

Reply:
> I think exactly like this and the previous comment, just not wanted to touch the existing code 😉.
But you decided to move the `stringUtils` into its own module inside this PR?

The color-conversion utils  should go in `engine.c` for now.

Reply:
> But you decided to move the stringUtils into its own module inside this PR? :P

This was done to prevent a cyclic dependency.